client/src/pages/people/Show.js

```diff
@@ -228,10 +228,21 @@ const PersonShow = ({ pageDispatchers }) => {
     <a href={`mailto:${person.emailAddress}`}>{person.emailAddress}</a>
   )
 
-  const orderedFields = orderPersonFields()
+  // Keys of fields which should span over 2 columns
+  const fullWidthFieldKeys = ["biography"]
```

Comment:
Perhaps keep this similar to `Compact.js` https://github.com/NCI-Agency/anet/blob/span-biography-over-2-columns/client/src/pages/people/Compact.js#L113

Reply:
Created `showAsFullWidthFields` property in the dictionary. 
It is an amendment to the `showPageOrderedFields`, which implies `biography` should be listed in both, which is probably not ideal. Yet the constency between `Show.js` and `Compact.js` is now there.